pkg/glance/pvc.go

```diff
@@ -16,7 +16,10 @@ func GetPvc(api *glancev1.GlanceAPI, labels map[string]string, pvcType PvcType)
 	var err error
 	requestSize := api.Spec.StorageRequest
 	pvcName := ServiceName
+	pvcAnnotation := map[string]string{}
+
 	if pvcType == "cache" {
```

Comment:
I know this wasn't changed in this PR, but perhaps could use `PvcCache` constant here instead?

Reply:
Makes sense, and done in the last PS